api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java

```diff
@@ -29,9 +38,11 @@
     private static final String INSUFFICIENT_INSTANCE_CAPACITY = "InsufficientInstanceCapacity";
     private static final String NODE_NAME = "node-1";
     private static final String NODE_ID = "node-id";
+    private static final String STUB_STRING = "";
```

Comment:
Could you please rename `STUB_STRING` to `TEST_TAG` with some value, to make test a little bit more clear.

Reply:
Done